moviebot/controller/controller_flask.py

```diff
@@ -0,0 +1,398 @@
+        response = (
+            "To start the conversation say Hi or Hello, or simply "
+            'enter you preferences ("I want a horror movie from the 90s").\n\n'
+            'To restart the recommendation process, issue "/restart".\n\n'
+            'To end the conversation, issue "/exit" or say Bye/Goodbye.\n\n'
+            'To see these instructions again, issue: "/help".'
+        )
+
+        return self.user_messages[user_id].text(
+            response, intent="REVEAL.DISCLOSE"
+        )
+
+    def store_data(self, user_id: str) -> HTTP_OBJECT_MESSAGE:
+        """Instructions for deleting stored conversation history.
+
+        Args:
+            user_id: User id.
+
+        Returns:
+            Object with instructions message to send to the server.
+        """
+        policy = (
+            'Type "/delete" at any time to stop storing and delete conversation'
+            " history.\n\nPress start to continue."
+        )
+        return self.user_messages[user_id].text(
+            policy, intent="REVEAL.DISCLOSE"
+        )
+
+    def first_time_message(self, user_id: str) -> HTTP_OBJECT_MESSAGE:
+        """Creates utterance with greetings message.
+
+        Args:
+            user_id: User id.
+
+        Returns:
+            Object with greetings message to send to the server.
+        """
+        self.start_agent(user_id)
+        greet = self.greeting()
+
+        return self.user_messages[user_id].text(
+            greet["message"]["text"], intent="REVEAL.DISCLOSE"
+        )
+
+    def privacy_policy(self, user_id: str = None) -> HTTP_OBJECT_MESSAGE:
+        """Creates utterance with policy.
+
+        Args:
+            user_id: User id. Defaults to None.
+
+        Returns:
+            Object with privacy message to send to the server.
+        """
+        title = (
+            "We may store some information to improve recommendations.\n"
+            "You may delete stored data at any time.\n"
+            "Read more in our privacy policy\n"
+            "Privacy Policy: https://iai-group.github.io/moviebot/Privacy_policy.html\n"  # noqa
+            'To accept the privacy policy write "/store"\n'
+            'To continue without accepting write "/continue"'
+        )
+        message = {"message": {"text": title, "intent": self.agent_intent}}
+        return message
+
+    def greeting(self) -> HTTP_OBJECT_MESSAGE:
+        """Creates greeting message.
+
+        Returns:
+            Object with greetings message to send to the server.
+        """
+        return {
+            "message": {
+                "locale": "default",
+                "text": "Hi there. I am IAI MovieBot, your movie recommending"
+                " buddy. I can recommend you movies based on your preferences."
+                "\nI will ask you a few questions and based on your answers, "
+                "I will try to find a movie for you.\n\n",
+                "intent": self.agent_intent,
+            }
+        }
```

Comment:
Consider formatting message in HTTPDataFormatter.

Reply:
Done.